cmd/zfs/zfs_main.c

```diff
@@ -3466,9 +3468,40 @@ print_header(list_cbdata_t *cb)
 			(void) printf("%-*s", (int)pl->pl_width, header);
 	}
 
+	color_end();
+
 	(void) printf("\n");
 }
 
+/*
+ * Decides on the color that the avail value should be printed in.
+ * > 80% used = yellow
+ * > 90% used = red
+ */
+static const char *
+color_available(zfs_handle_t *zhp)
```

Comment:
Maybe call this something like `zfs_list_avail_color()`.  `color_available()` sounds a little too generic, like you're checking to see if a color is available in the terminal.

Reply:
Yep, that's a much better name, changed.